src/presentation/http/router/note.test.ts

```diff
@@ -50,3 +50,62 @@ describe('Note API', () => {
     });
   });
 });
+
+describe('Note API', () => {
+  describe('GET note/:notePublicId ', () => {
+    test('Returns note by public id', async () => {
+      const expectedStatus = 200;
+      
+      const expectedNote = {
+        'id': 2,
+        'publicId': 'testnote11',
+        'creatorId': 1,
+        'content': null,
+        'createdAt': '2023-10-16T13:49:19.000Z',
+        'updatedAt': '2023-10-16T13:49:19.000Z',
+      };
+
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        url: '/note/testnote11',
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      const body = response?.body !== undefined ? JSON.parse(response?.body) : {};
+
+      expect(body).toStrictEqual(expectedNote);
+    });
+
+    test('Returns 403 when permission denied', async () => {
+      const expectedStatus = 403;
+
+      const response = await global.api?.fakeRequest({
+        method: 'GET',
+        url: '/note/testnote22',
+      });
+
+      expect(response?.statusCode).toBe(expectedStatus);
+
+      const body = response?.body !== undefined ? JSON.parse(response?.body) : {};
+
+      expect(body).toStrictEqual({ message: 'Permission denied' });
+    });
+
+    test('Returns 406 when public id incorrect', async () => {
```

Comment:
describe what does it mean "incorrect"

Reply:
> describe what does it mean "incorrect"

Do I understand correctly that the 406 status can only be caused by incorrect characters within the id?

Reply:
> describe what does it mean "incorrect"

I rewrote the description, check please